colossalai/utils/model/utils.py

```diff
@@ -3,9 +3,9 @@
 from typing import Optional
 
 
-def _substitute_init_recursively(cls, func):
+def substitute_init_recursively(cls, func):
```

Comment:
Not sure changing this API will affect other contexts.

Reply:
I have already searched in our repo, and change them to fit the new api.